### ISSUE TITLE
Status cover pages: "pre-final draft" stages

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-nist#99]
(UPDATED 2019/04/20 Public Draft layout updated, Added "Original Release Date" to preliminary, WIP and public draft)

This supersedes #91 (it got too long) with final information ready to implement.

NIST Draft stages are described in: https://github.com/metanorma/metanorma-model-nist/issues/16

A NIST "Draft" will use these cover pages to indicate their stages.

NOTE: All of these Draft covers use an identical structure! 🎆 

This cover page will replace the existing cover -- there will be no authors mentioned and no horizontal lines.

All Word cover pages are provided in `mn-nist-documents/references`.

The "warning texts" here should be implemented in some YAML or AsciiDoc form so we don't repeat in HTML/Word.

Internal draft:
<img width="508" alt="internal-draft" src="https://user-images.githubusercontent.com/11865/56454934-56da4c00-6393-11e9-8f38-577857a713a9.png">

Work in progress draft:

<img width="511" alt="wip-draft" src="https://user-images.githubusercontent.com/11865/56454936-59d53c80-6393-11e9-8184-45c866f1a111.png">

Preliminary draft:

<img width="510" alt="preliminary-draft" src="https://user-images.githubusercontent.com/11865/56454938-5d68c380-6393-11e9-8613-ba3532fa268d.png">

Public drafts:

* "Initial Public Draft (IPD)" is changed to "Second Public Draft (2PD)", "Third Public Draft (3PD)", ... "Final Public Draft (FPD)" as appropriate.
* The DOI link and "This publication is available at..." wording only appears when a DOI has been provided.

<img width="507" alt="public-draft" src="https://user-images.githubusercontent.com/11865/56454941-62c60e00-6393-11e9-9a25-7368460fdf03.png">

## Valid states for Pre-final Drafts

"Pre-final Drafts":
* Internal Draft
* WIP Draft
* Preliminary Draft
* Public Draft (IPD, 2PD, 3PD, ..., FPD)

Valid states for Pre-final Drafts:
* Active (default)
* Withdrawn (add "Withdrawn Draft" cover on top of existing stage cover)
* Retired (add "Retired Draft" cover on top of existing stage cover)

When applying a status page cover for documents at "pre-final states":
* Use the Draft Cover page first, followed immediately by the Title page

**Comment by ronaldtse** [comment]
Public draft cover updated:
> Fixed a couple of typos in the warning statement. Next-to-last sentence should end “…are necessary.” In the last sentence, updated it to read: …will not be considered “final” until it is formally…

[pre-final-draft-covers.pdf](https://github.com/metanorma/metanorma-nist/files/3099890/pre-final-draft-covers.pdf)

<img width="507" alt="Screen Shot 2019-04-20 at 5 38 13 PM" src="https://user-images.githubusercontent.com/11865/56454922-301c1580-6393-11e9-8d9c-0e1c8cb97313.png">

WARNING: Screenshots in original post updated to show "Original release date" (except Internal Draft)

**Comment by opoudjis** [comment]
Cannot close this ticket until https://github.com/metanorma/metanorma-nist-style/issues/4 is done for HTML.

**Comment by opoudjis** [comment]
The original documents use text boxes, because they do not give a crap about reuse. Text boxes in turn mean that I have to hard-code their height. I will of course do no such thing, and will convert them to normal bordered divs. I may hard code the position of the banner, at most.

**Comment by opoudjis** [comment]
Fixed banner for drafts to 180pt from bottom.

**Comment by opoudjis** [comment]
The Public Draft is too tight for comfort. (Of course, this entire exercise is pointless to begin with, and that tightness confirms as much.) I've removed one of the two carriage returns between the Stage and the Series Number: the minute there's a subtitle, otherwise, the banner will end up appearing above the DOI.

**Comment by ronaldtse** [comment]
> The Public Draft is too tight for comfort. (Of course, this entire exercise is pointless to begin with, and that tightness confirms as much.) I've removed one of the two carriage returns between the Stage and the Series Number: the minute there's a two-line subtitle, otherwise, the banner will end up appearing above the DOI.

That's fine. The key point is to have all the cover pages uniform in their implementation.

Thanks!

**Comment by opoudjis** [comment]
Crap, can't close this ticket, it's waiting on the HTML pages. Reopening.

**Comment by opoudjis** [comment]
HTML done. @ronaldtse, if you have concerns about the HTML pages I've sent you, please raise a new ticket.
